playground/src/style.css

```diff
@@ -0,0 +1,60 @@
+* {
+  box-sizing: border-box;
+}
+
+body,
+html,
+#root,
+#app {
+  margin: 0;
+  height: 100%;
+  width: 100%;
+}
+
+#app {
+  display: flex;
+}
+
+.options {
+  height: 100vh;
+  overflow-y: scroll;
+  padding: 1em;
+  min-width: 300px;
+  border-right: 1px solid lightgray;
+}
```

Comment:
I might spend some time re-styling this, to make the UI more cohesive between the settings pane and the editor. I think the settings should have some kind of show/hide UX too -- perhaps they should even be hidden by default.

Reply:
E.g., something like this would make sense.

<img width="1624" alt="Screen Shot 2022-12-23 at 11 29 04 PM" src="https://user-images.githubusercontent.com/1309177/209421206-05dcdf01-2436-4bf2-ab49-fc28493e292c.png">

Reply:
Thank you, that sounds reasonable. Styling isn't my strong skill. 

Just let me know when you want to tackle that, I'll address your other comments for now.

Reply:
There's also lots of potential in making the user input nicer, based on the type of the option. E.g. presenting checkboxes for booleans. I didn't go into that to keep the PR smallish.